.docker/src-www/etc/nginx/partials/www.conf.tmpl

```diff
@@ -0,0 +1,27 @@
+##################################################
+
+server {
+    listen 80;
+
+    server_name ${WWW_HOST} _;
+
+    include /etc/nginx/partials/www_common.conf;
+
+    # Only www.usa.gov should be indexed.
+    add_header X-Robots-Tag noindex;
+}
+
+server {
+    listen 443 ssl;
+    include partials/ssl.conf;
+
+    server_name ${WWW_HOST} _;
+
+    include /etc/nginx/partials/www_common.conf;
+
+    # Only www.usa.gov should be indexed.
+    add_header X-Robots-Tag noindex;
```

Comment:
The `X-Robots-Tag: noindex` header needs to be added for all sites except www.usa.gov. I think the answer is just to remove it here. However, I need to verify how it is that we're adding this to the static sites other than www.usa.gov.

Reply:
Here we go! In the version of this file that exists in src-cms on the dev branch, there are two things we need to maintain here (unless they are somewhere else that I missed): 

1. Separate blocks for the www and beta hosts, versus beta-stage and beta-dev. The www block lacks the X-Robots header
2. The beta block, with a redirect to www. 

IMHO we should be able to take the version from dev verbatim, but the fact that you changed it gives me pause. Let's chat!

Reply:
I believe you are correct.  Pasting from dev branch, and deploying to dev env.